edsdme/blocks/search/SearchCards.js

```diff
@@ -49,31 +48,79 @@ export default class Search extends PartnerCards {
       </div>`;
   }
 
-  setContentTypeCounts() {
-    const counts = { countAll: 0, countAssets: 0, countPages: 0 };
-    const filteredCards = [];
+  // eslint-disable-next-line consistent-return
+  async getCards() {
+    const url = new URL(
+      'https://14257-dxpartners-stage.adobeioruntime.net/api/v1/web/dx-partners-runtime/search-apc/search-apc?',
```

Comment:
@SonjaPopovic we could already swap the domain for [io-partners-dx.stage.adobe.com](http://io-partners-dx.stage.adobe.com/)

we will also need to add a switch here to check the environment and use the prod runtime domain, although there are no prod open search indexes currently so the action won't work.